File: Java/백준/Bronze/1977. 완전제곱수/완전제곱수.java
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int sum = 0;
        int min = 0;
        for (int i=a;i<=b;i++){
            if (Math.pow(i,0.5)%1==0){
                sum += i;
                if (min==0){
                    min = i;
                }
            }
        }
        if (sum==0){
            System.out.println(-1);
        }
        else{
            System.out.println(sum);
            System.out.println(min);
        }
    }
}

File: C#/백준/Bronze/1977. 완전제곱수/완전제곱수.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int b = int.Parse(Console.ReadLine());
            int min = 10001;
            int sum = 0;
            for (int i=a;i<=b;i++){
                if(Math.Pow(i,0.5)%1==0){
                    sum+=i;
                    min = min<i?min:i;
                }
            }
            if (sum==0){
                Console.WriteLine(-1);
            }
            else{
                Console.WriteLine(sum);
                Console.WriteLine(min);
            }
        }
    }
}